### DISCUSSION TITLE
How conditional competition specifies requests

### DISCUSSION THREAD

**Discussion by M00nBack** [projectdiscovery/nuclei#4250] [Q&A]
I have two request packages to send. The first request is to extract the csrf token. The second request requires multiple requests for conditional competition. How should I write the template?

**Comment by ehsandeep** [comment]
See example template - https://templates.nuclei.sh/public/servicenow-widget-misconfig
See docs - https://docs.nuclei.sh/template-guide/operators/extractors#dynamic-extractor

**Reply by M00nBack** [reply]
race: true
    race_count: 200


I use this but @once doesn't seem to work, plus I need to use unsafe: true to send the original http packet

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
